### PR TITLE
build!: don't ship untranspiled code or commit transpiled code

### PR DESCRIPTION
**Pull request by RyanZim**
After #3

### CODE REVIEW

**Comment by mvayngrib** [comment]
tests are failing cause make-dir-cli is an ESM module which node 6 doesn't like

**Review by mvayngrib — APPROVED** [review]
utACK

**Comment by RyanZim** [comment]
Rebased on #3; tests should pass now.

File: package.json [hunk 1]
```diff
@@ -9,13 +9,17 @@
...
+  "files": [
+    "src/"
+  ],
   "repository": {
     "type": "git",
     "url": "git+https://github.com/ExodusMovement/buffer-noise.git"
   },
   "scripts": {
-    "build": "flow-remove-types ./src/index.js > ./lib/index.js",
+    "build": "make-dir lib && flow-remove-types ./src/index.js > ./lib/index.js",
```
Comment (fboucquez) [inline package.json:20]: If we are not shipping transpiling code, why are we building? Should lint be enough to detect any issue babel can detect? Can we remove babel?

There are general questions, no just for this PR

**Pull request merged**